canonical-websites/snapcraft-flask
# Sanitise and linkify markup

**Proposal**. I don't know if we want this feature, but here's an example of how we might handle markup and URLs in snap descriptions.

This will:

- Remove any dangerous tags (e.g. `<script>`) from descriptions
- Allow safe markup tags (e.g. `<strong>`) in descriptions, and render them properly
- Add `<a rel="nofollow">` tags to URLs found in text
- Add `rel="nofollow"` to existing link tags

QA
--

`./run`, then visit <http://0.0.0.0:8022/dwarf-fortress/> and see the links are clickable! I'm not sure if there's an example of a snap with markup in its description, but if one exists we could try it out as well.
